test/helpers/fixture_deploy_helper.rb

```diff
@@ -51,12 +51,14 @@ def deploy_dir(dir, wait: true, allow_protected_ns: false, prune: true, bindings
       current_sha: SecureRandom.hex(6),
       context: KubeclientHelper::MINIKUBE_CONTEXT,
       template_dir: dir,
-      wait_for_completion: wait,
-      allow_protected_ns: allow_protected_ns,
-      prune: prune,
+      logger: test_logger,
       bindings: bindings
     )
-    runner.run
+    runner.run(
+      skip_wait: !wait,
```

Comment:
The thing that triggered this change was that this option becomes a local variable in `run` that had the same name as the actual `wait_for_completion` method (that does the waiting), which was confusing. I reached for this name to match the CLI flag (`opts.on("--skip-wait") { skip_wait = true }`), but I could change it to just `wait` or maybe `verify_result` instead.